MatrixSDK/MXRestClient.m

```diff
@@ -4014,6 +4067,46 @@ - (MXHTTPOperation*)uploadKeys:(NSDictionary*)deviceKeys oneTimeKeys:(NSDictiona
                                  }];
 }
 
+- (MXHTTPOperation*)uploadKeys:(NSDictionary*)deviceKeys oneTimeKeys:(NSDictionary*)oneTimeKeys
+               forDeviceWithId:(NSString*)deviceId
```

Comment:
this parameter doesn't seem to be used anywhere.  It should be included in the path as `/keys/upload/<deviceId>` (after URL-encoding the device ID).